pkgs/development/tools/parsing/tree-sitter/update.nix

```diff
@@ -55,6 +56,8 @@ let
     "ruby-tree-sitter.old"
     # this is the (unmaintained) rust language bindings, tree-sitter-rust is the grammar
     "rust-tree-sitter"
+    # this is the kotlin language bindings, tree-sitter-kotlin is the grammar
+    "kotlin-tree-sitter"
```

Comment:
This belongs in a different commit.

Reply:
What do you mean by that? That I somehow took it from another commit (if it is the case, sorry must have been during conflict resolution), or that it should be the object of an individual commit?

Reply:
Just another individual commit specific to this action of adding a new grammar.

Reply:
Thank you for the clarification.

I don't want to rebase now because it will make the MR less readable. 

But I can definitely do that just before merging.

Reply:
> I don't want to rebase now because it will make the MR less readable.

I disagree, it makes it easier to review if you have atomic commits.

Reply:
Ok, I will rebase then.

Reply:
Well, tried to change the history based on your suggestions. I am not sure it's clearer :sweat_smile: